packages/css/src/rules/types.ts

```diff
@@ -211,6 +212,21 @@ export type RecipeVariants<
   >
 > = RulesVariants<RecipeFn>;
 
+export type SlotRecipeStyleRule = {
+  base?: RecipeStyleRule | CSSPropertiesWithVars;
+  variants?: VariantGroups;
+  toggles?: VariantDefinitions;
+} & (CSSRule | CSSPropertiesWithVars);
+
+export type SlotRecipeDefinition = Record<string, SlotRecipeStyleRule>;
+
+export type SlotRecipeResult<Slots extends SlotRecipeDefinition> = {
+  [Slot in keyof Slots]: RuntimeFn<
+    ConditionalVariants<Slots[Slot]["variants"], Slots[Slot]["toggles"]>,
+    ComplexPropDefinitions<PropTarget>
+  >;
+};
```

Comment:
How about use [MultipleRecipe](https://github.com/mincho-js/working-group/blob/main/text/004-methods.md#3-multiple)?
As we improve the API, `cssVariant` and `rulesVariant` will be changed to the concept of mutiple.

I hope the terminology in the above code is also unified appropriately!!